### PR TITLE
main: handle errors

### PR DESCRIPTION
**Pull request by davidvgalbraith**
Add an error-handler div to the HTML and make a JCL Errors
object that writes to it. Call errors.render on any errors
or warnings

fixes https://github.com/juttle/outrigger/issues/29

@mnibecker @go-oleg

### CODE REVIEW

**Comment by davidvgalbraith** [comment]
compile-time error (`read file` with no file):
compile time

**Comment by davidvgalbraith** [comment]
Runtime error (`read file -file "nobody"`):
runtime

File: gulpfile.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ function gulp_test() {
     return gulp.src('test/**/*.spec.js')
         .pipe(mocha({
             log: true,
-            timeout: 5000,
+            timeout: 30000,
```
Comment (rlgomes) [inline gulpfile.js]: Are tests timing out for you ?

Comment (davidvgalbraith) [inline gulpfile.js]: Yeah!

Comment (davidvgalbraith) [inline gulpfile.js]: You can see it in my failed build: `✓ Start 2 jobs at once, should return both jobs (8150ms)` would have failed without this change.

File: src/apps/run/main.js [hunk 2]
```diff
@@ -22,10 +23,22 @@ let initBundle = (bundle) => {
...
+                        errors.render(warning);
+                    });
+                });
         } else {
-            inputs.render(bundle);
+            return inputs.render(bundle);
         }
+    })
+    .catch((err) => {
+        return errors.render(err);
```
Comment (mnibecker) [inline src/apps/run/main.js]: I think we need a `.catch` in the button render method

**Comment by mnibecker** [comment]
cool looks good, though we should probably wait until we have cut-release on the depending feature in `juttle-client-library` before we merge this.

**Pull request merged**